src/openms/source/CONCEPT/QCMS2IdentificationRate.cpp

```diff
@@ -0,0 +1,52 @@
+#include <OpenMS/KERNEL/StandardTypes.h>
+#include <OpenMS/CONCEPT/QCMS2IdentificationRate.h>
+#include <OpenMS/FORMAT/IdXMLFile.h>
+#include <OpenMS/FORMAT/MzMLFile.h>
+#include <OpenMS/FORMAT/FeatureXMLFile.h>
+#include <boost/regex.hpp>
+#include <OpenMS/FORMAT/PercolatorOutfile.h>
+
+
+using namespace OpenMS;
+using namespace std;
+
+QCMS2IdentificationRate::~QCMS2IdentificationRate(){
+
+}
+
+int QCMS2IdentificationRate::MS2IDRateidentifier_( MzTabFile& MzTabOutputFile, String out)
+{
+  MzTab mztab;
+  vector<String> idXMLFiles;
+  Size control;
+  vector<String> irawfiles;
+  boost::regex rgx("[A-Za-z0-9]+[.]idXML");
+  boost::regex rgx2(".idXML");
+  for(vector<pair<String,String>>::const_iterator it = ivec_.begin();it!=ivec_.end();++it)
+  {
+    if(it->first=="Post_FalseDiscoveryRate"){
```

Comment:
why do you filter here? The files given in the Constructor can be expected to be correct. this class should not need to know about `Post_FalseDiscoveryRate`